tests/components/jewish_calendar/test_binary_sensor.py

```diff
@@ -1,17 +1,16 @@
 """The tests for the Jewish calendar binary sensors."""
-from datetime import timedelta
-from datetime import datetime as dt
+from datetime import datetime as dt, timedelta
 
 import pytest
 
-from homeassistant.const import STATE_ON, STATE_OFF
-import homeassistant.util.dt as dt_util
-from homeassistant.setup import async_setup_component
 from homeassistant.components import jewish_calendar
+from homeassistant.const import STATE_OFF, STATE_ON
+from homeassistant.setup import async_setup_component
+import homeassistant.util.dt as dt_util
 
-from tests.common import async_fire_time_changed
-from . import alter_time, make_nyc_test_params, make_jerusalem_test_params
```

Comment:
Not really sure this is correct, importing tests.common after "."

Usually we want the most "local" import to be last. Python built-in, third-party, package (home assistant), tests, and then local to the module?

Reply:
Fixed, thanks for paying attention!